src/System.Net.Sockets/src/System/Net/Sockets/NetworkStream.cs

```diff
@@ -568,7 +549,7 @@ protected override void Dispose(bool disposing)
                             if (chkStreamSocket != null)
                             {
                                 chkStreamSocket.InternalShutdown(SocketShutdown.Both);
-                                chkStreamSocket.Close(_closeTimeout);
```

Comment:
I presume `_closeTimeout` is not used anymore?

As future plans: we should in the future include a `CloseAsync(int timeout)` API to re-enable this scenario.